### DISCUSSION TITLE
Customize Emails

### DISCUSSION THREAD

**Discussion by syntax53** [inventree/InvenTree#6672] [Q&A]
Hello,

Just _finally_ got InvenTree installed via the Bare Metal guide.  When I receive an email for email verification it says, 
"[server.fqdn.address] Please Confirm Your Email Address" in subject as well as the following in the body:

> Hello from server.fqdn.address!
> 
> You're receiving this email because user msnead has given your email address to register an account on server.fqdn.address.
> 
> To confirm this is correct, go to https://server.fqdn.address/accounts/confirm-email/code123abc/
> 
> Thank you for using server.fqdn.address!
> server.fqdn.address

Where are these pulling from and can they be changed?

Thanks

**Comment by matmair** [comment]
They are defined in the plugin [django-allauth here](https://github.com/pennersr/django-allauth/blob/0c0dbc2af82601f74547b87b5ca0fcfaef3ae029/allauth/templates/account/email/email_confirmation_message.txt) and customizing them is possible but requires Python, Django and InvenTree-specific knowledge.

**Reply by syntax53** [reply]
Thanks for the response.   What about simply changing the server's dns name to something more user friendly?  I don't see any mention of this plugin in the plugin settings.

**Comment by SchrodingersGat** [comment]
@syntax53 I have pushed a new PR which should make this simpler - https://github.com/inventree/InvenTree/pull/6673

**Reply by syntax53** [reply]
Hello and thank you for the update.  I haven't tried it yet and am a bit afraid to try and update the system after having worked so hard to get it running :) .  Looking at the change in code, it looks like that fix is to make it start using the parameter defined in the config.yaml for the email prefix?  If so, that addresses at least one of the four places it's referenced.  Will that also flow into the fields for the greeting and closing in the body?   i.e. "Hello from server.fqdn.address!" and "Thank you for using server.fqdn.address!"

**Reply by SchrodingersGat** [reply]
@syntax53 not sure - can you provide the code locations where those messages are generated?

**Reply by syntax53** [reply]
Sorry, I am super new to this application and only started learning about it at all just recently.  Limped through the "bare metal"  installation (dependency issues) for the first time just a few days ago.

**Reply by SchrodingersGat** [reply]
Looking through the code for `django-allauth` I find these messages:

```
{% load i18n %}{% autoescape off %}{% blocktrans with site_name=current_site.name %}Hello from {{ site_name }}!{% endblocktrans %}

{% block content %}{% endblock content %}

{% blocktrans with site_name=current_site.name site_domain=current_site.domain %}Thank you for using {{ site_name }}!
{{ site_domain }}{% endblocktrans %}
{% endautoescape %}
```

I am not sure whether this will be covered by the patch. If you do still see the wrong messages after updating, please let me know
